scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -29,6 +31,42 @@
   </li>
 {% endmacro %}
 
+{% macro alignments(institute, case, variant, current_user, case_groups, config) %}
+   <ul class="list-group">
+        <li class="list-group-item d-flex justify-content-between">
+          <div>
+            {{ igv_button(case, variant, case_groups) }}
+          </div>
+          <div>
+            {% if variant.alamut_link and config.SHOW_ALAMUT_LINK %}
```

Comment:
I wasn't thinking that a variant might also not have an alamut link. Isn't SHOW_ALAMUT_LINK superfluous then? 🤔 

Otherwise buttons look good, well aligned:
![image](https://user-images.githubusercontent.com/28093618/130737483-eb9966f8-df6e-45d3-ba9d-893e3022da45.png)

Reply:
Ah, I thought you had already tried putting it in links and realised it was looking a bit lonely there. The thing is the link will be made if it can be made, regardless if the user has alamut or not. So the option is still good.

Reply:
Right! Then I think it's good to merge!

Reply:
Meh, I just realised we do have a config for links: the external primer order. 🙄
Anyway, it is like two mentions. We can move it there later.

Reply:
Oh, actually, if we do that: wouldn't it be cleaner to introduce a HIDE_ALAMUT_LINK? Then doing nothing will just produce the link as before. I´ll PR and be done with it while we are still on the same version..

Reply:
Agreed!